grafana/Dockerfile

```diff
@@ -16,14 +16,10 @@ RUN curl -sSLf https://deb.nodesource.com/setup_16.x | bash - \
     && rm -rf /var/lib/apt/lists/* \
     && npm install --global yarn
 
-# `--network-concurrency 1` is a (possibly incomplete) workaround for
-# "Extracting tar content of undefined failed, the file appears to be corrupt" random error
-RUN yarn install --pure-lockfile --no-progress --network-concurrency 1
-ENV NODE_ENV production
-RUN yarn build
+# This suppresses a minor error message
+RUN mkdir -p plugins-bundled/external
 
-RUN go mod verify
-RUN go run build.go build
+RUN make deps-js gen-go deps-go build-js build-go
```

Comment:
I'm not sure why you take a different way from the upstream Dockerfile.

Reply:
This modification is originally due to my misunderstanding. I will follow the original process.